distributed/comm/ucx.py

```diff
@@ -37,13 +37,13 @@
 device_array = None
 
 
-def synchronize_stream(stream=0):
+def synchronize_stream(comm=0):
     import numba.cuda
 
     ctx = numba.cuda.current_context()
-    cu_stream = numba.cuda.driver.drvapi.cu_stream(stream)
-    stream = numba.cuda.driver.Stream(ctx, cu_stream, None)
-    stream.synchronize()
+    cu_stream = numba.cuda.driver.drvapi.cu_stream(comm)
+    comm = numba.cuda.driver.Stream(ctx, cu_stream, None)
+    comm.synchronize()
```

Comment:
This is actually a different kind of stream (CUDA streams rather than Dask streams) and we should probably leave it as is.